empchat/datasets/empchat.py

```diff
@@ -117,9 +117,49 @@ def __init__(
         self.data = []
         self.ids = []
         history = []
+        cands = []
+        from allennlp.models.archival import load_archive
+        from allennlp.service.predictors import Predictor
+
+        #archive = load_archive("https://s3-us-west-2.amazonaws.com/allennlp/models/elmo-constituency-parser-2018.03.14.tar.gz", cuda_device=0)
+        archive = load_archive("elmo-constituency-parser-2018.03.14.tar.gz", cuda_device=0)
```

Comment:
@kohlisimranjit Do we need add configuration parameter to turn on/off constituency parser? This way, we can test with and without this features to do comparison test without changing code.